src/GitHubActions.jl

```diff
@@ -246,6 +246,8 @@ function Logging.handle_message(
             "warning"
         elseif level === Error
             "error"
+        else
+            "unknown"
```

Comment:
Maybe one could use
```suggestion
            "notice"
```
here? AFAICT based on https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-a-debug-message and the sections thereafter, Github actions don't understand `::unknown ...` but there is support for annotations with `::notice ...` which follow the same format as `::error ...` etc. (so I assume it should just work).

Reply:
That seems reasonable